packages/playwright/src/tests/upload-arts.spec.ts

```diff
@@ -0,0 +1,168 @@
+import { faker } from '@faker-js/faker/locale/en'
+import { expect, test } from '@playwright/test'
+
+import { PASSWORD, USERNAME } from '../constants'
+import { HomePage, LoginPage } from '../pages'
+import { ArtsPage } from '../pages/Arts'
+import { DashboardArtsPage } from '../pages/Dashboard'
+import { ProfilePage } from '../pages/Profile'
+import { getVercelUrl } from '../utils'
+
+test.afterEach(async ({ page }) => {
+  await page.close()
+})
+test.describe('Upload Arts', () => {
+  test('TC01- should not upload art without logging in', async ({ page }) => {
+    const homePage = new HomePage(page, 'kunsthalte')
+    const artsPage = new ArtsPage(page)
+    await page.goto(getVercelUrl('kunsthalte'))
+
+    await homePage.clickArtsMenu()
+    await artsPage.clickOnTheUploadsArtButton()
+    expect(artsPage.warning).toContainText(
+      'You must be logged in in order to be able to upload an art!',
+    )
+  })
+
+  test('TC02- should upload art with logging in', async ({ page }) => {
+    const loginPage = new LoginPage(page)
+    const homePage = new HomePage(page, 'kunsthalte')
+    const artsPage = new ArtsPage(page)
+
+    await page.goto(homePage.url, { waitUntil: 'domcontentloaded' })
+    await homePage.gotoLogin()
+    await loginPage.login(USERNAME, PASSWORD)
+    await homePage.clickArtsMenu()
+    await artsPage.clickOnTheUploadsArtButton()
+
+    await page.waitForTimeout(2000)
+    expect(artsPage.titleInput).toBeVisible()
+  })
+
+  test('TC03- should fill required fields for upload art', async ({ page }) => {
+    const loginPage = new LoginPage(page)
+    const homePage = new HomePage(page, 'kunsthalte')
+    const artsPage = new ArtsPage(page)
+
+    await page.goto(homePage.url, { waitUntil: 'domcontentloaded' })
+    await homePage.gotoLogin()
+    await loginPage.login(USERNAME, PASSWORD)
+    await page.waitForTimeout(1000)
+
+    await homePage.clickArtsMenu()
+    await artsPage.clickOnTheUploadsArtButton()
+    await page.waitForTimeout(1000)
+    const filePath = 'apps/kunsthalte/public/arts.jpeg'
+    await artsPage.uploadButton.setInputFiles(filePath)
```

Comment:
This block is being used multiple times. Wouldn't be nicer to move this steps into a new function in `Arts.ts` page.




```suggestion
    await artsPage.uploadImage(imagePath: string)
```
___

I would define a variable (`IMAGE_PATH`) in constants.ts for `filePath`. Other testers can use the same image in the future.